auth/api/iam/api.go

```diff
@@ -953,6 +991,13 @@ func (r Wrapper) accessTokenServerStore() storage.SessionStore {
 	return r.storageEngine.GetSessionDatabase().GetStore(accessTokenValidity, "serveraccesstoken")
 }
 
+var oauthRequestObjectKey = []string{"oauth", "requestobject"}
```

Comment:
can we put these together somewhere?

Reply:
moved to the top of the file to match `openid4vp.go` and `user.go`. 

Another option would be to move all session stores to a separate file? We currently have 10.